pkg/prometheusexporter/prometheus_exporter.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Create `/pkg/exporters/` and put both k8s and prometheus exporters into it.

Reply:
Done.